pages/create.js

```diff
@@ -99,9 +99,8 @@ const StyledSuccessMessage = styled.h5`
   border-radius: 8px;
 `;
 
-// -------------- FORM FUNCTION------------
-// ------------------here------------------
-// ------------------DOWN------------------
+//----------------------------------------------- FUNCTION------------HERE
+
 
 export default function CreateContactForm() {
   const [showSuccessMessageState, setShowSuccessMessageState] = useState(false);
```

Comment:
Below this state is some outcommented code that should be removed. :)

And below there in your `StyledContainer` is a typo: `Notizen` instead of `Notitzen`.

Reply:
changed